modules/eventListeners.js

```diff
@@ -0,0 +1,30 @@
+const seeListsOfBooks = document.getElementById('list-of-books');
+const addNewBook = document.getElementById('add-new-link');
+const contactLink = document.getElementById('contact-link');
+
+export default function seeBooks() {
+  seeListsOfBooks.classList.add('active');
+  addNewBook.classList.remove('active');
+  contactLink.classList.remove('active');
+  document.getElementById('awesome-books').classList.remove('hide');
+  document.getElementById('add-book').classList.add('hide');
+  document.getElementById('contact-us').classList.add('hide');
+}
+
+export function addToBooks() {
```

Comment:
- [ ] Kindly refactor to an arrow function.

Reply:
Resolved. Thanks